Automattic/woocommerce-payments
# Remove unneeded code from PO experiment

Partially Fixes server/4566

#### Changes proposed in this Pull Request





This PR cleans up some code which is no longer used. It was part of the first Progressive Onboarding experiments, but ended up not making it into the MVP. I will follow up with a server PR deprecating the server endpoints.





#### Testing instructions





* Onboard a new account, verify you don't see any console errors or failed requests during the MOX flow.

-------------------


**Post merge**
